src/drawing/drawingUtil.ts

```diff
@@ -0,0 +1,34 @@
+import { Annotation, AnnotationDrawing, PathGroup, Position, Shape } from '../@types';
+
+export function getCenter({ height, width, x, y }: Shape): Position {
+    return { x: x + width / 2, y: y + height / 2 };
+}
+
+export function getShape(pathGroups: PathGroup[]): Shape {
+    let maxX = 0;
+    let maxY = 0;
+    let minX = Number.MAX_VALUE;
```

Comment:
Should this be `maxX`, since it's related to MAX_VALUE?

Reply:
We will run `Math.min(minX, x)` so the initial value should be a very large number. Will add a comment about this
```
    // Initialize mins with MAX_VALUE so that any possible
    // input value of the first point can be captured by Math.min
```